students/nam_vo/lesson03/list_lab.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+
+def series1():
+    # Create a list that contains “Apples”, “Pears”, “Oranges” and “Peaches”
+    l = ['Apples', 'Pears', 'Oranges', 'Peaches']
+    # Display the list
+    print(l)
+
+    # Ask the user for another fruit and add it to the end of the list
+    fruit = input('Enter another fruit > ')
+    l.append(fruit)
+    # Display the list
+    print(l)
+
+    # Ask the user for a number and display the number back to the user and the fruit corresponding to that number (on a 1-is-first basis)
+    number = input('Enter a number > ')
+    print(f'{number}. {l[int(number) - 1]}')
+
+    # Add another fruit to the beginning of the list using “+” and display the list
+    l = ['Watermelon'] + l
+    print(l)
+
+    # Add another fruit to the beginning of the list using insert() and display the list
+    l.insert(0, 'Kiwi')
+    print(l)
+
+    # Display all the fruits that begin with “P”, using a for loop
+    for item in l:
+        if item[0] == 'P':
+            print(item)
+
+    return l
+
+def series2(a_list):
+    # Display the list
+    print(a_list)
+
+    # Remove the last fruit from the list and display the list
+    a_list.pop()
+    print(a_list)
+
+    # Ask the user for a fruit to delete, find it and delete it
+    fruit = input('Enter a fruit name > ')
```

Comment:
Might want to say 'Enter a fruit name to remove it from the list' or similar.  More explicit